{{cookiecutter.project_name}}/noxfile.py

```diff
@@ -254,9 +261,22 @@ def xdoctest(session: Session) -> None:
     session.run("python", "-m", "xdoctest", package, *args)
 
 
+@nox.session(name="docs-build", python="3.8")
+def docs_build(session: Session) -> None:
+    """Build the documentation."""
+    args = session.posargs or ["docs", "docs/_build"]
+    install(session, "sphinx")
```

Comment:
We also need to install the package for `autodoc`:

```suggestion

    install_package(session)
    install(session, "sphinx")
```

Reply:
I will do that, to be honest, I've read the comments of `install_package` and `install` and got a bit confused of why some sessions use both. But I also confess I didn't spend much time trying to understand =p